arkouda/client.py

```diff
@@ -523,9 +535,71 @@ def shutdown() -> None:
     serverConfig = None
 
 
+def _json_args_to_str(json_obj: Dict) -> Tuple[int, str]:
+    """
+    Convert Python Dictionary into a JSON formatted string that can be parsed by the msg
+    processing system on the Arkouda Server
+
+    Parameters
+    ----------
+    json_obj : dict
+        Python dictionary of key:val representing command arguments
+
+    Return
+    ------
+    Tuple - the number of parameters found and the json formatted string
+
+    Raises
+    ------
+    TypeError
+        - Keys are a type other than str
+        - Any value is a dictionary.
+        - A list contains values of multiple types.
+
+    Notes
+    -----
+    - Nested dictionaries are not yet supported, but are planned for future support.
+    - Support for lists of pdarray or Strings objects does not yet exist.
+    """
+    from arkouda.pdarrayclass import pdarray
+    from arkouda.strings import Strings
+
+    j = []
+    for key, val in json_obj.items():
+        if not isinstance(key, str):
+            raise TypeError(f"Argument keys are required to be str. Found {type(key)}")
+        if isinstance(val, dict):
+            raise TypeError("Nested JSON is not currently supported for server messages.")
+        param = {"key": key}
+
+        if isinstance(val, pdarray):
+            param["objType"] = "pdarray"
```

Comment:
would not be a bad idea to replace with an enum here that can be passed to the server side, but good for now

Reply:
My brain may be failing me today, but I am not sure what the enum would really buy us here. @hokiegeek2 can you expand on why you'd like to see an enum here for my understanding? Thanks!

Reply:
@Ethan-DeBandi99 it would prevent a developer from mistyping an an objType string. When the string is passed into the Enum and is wrong, a ValueError is raised